.gitignore

```diff
@@ -62,4 +62,7 @@ typings/
 # dotenv environment variables file
 .env
 ext_resources/
-build/
```

Comment:
@hereisnaman I think you deleted this by mistake. Actually we need this in .gitignore. :)

Reply:
I think you need to recheck that it is the first rule in the file. It was repeated at the bottom. So i removed one.